radar_control/radar_control.py

```diff
@@ -20,11 +20,16 @@
 import os
 import zmq
 import pickle
+import threading
+import math
+import numpy as np
+
 from functools import reduce
 
 sys.path.append(os.environ["BOREALISPATH"])
```

Comment:
I agree. I am in favour of doing it all as below, and avoiding `sys.path.append` altogether.

Reply:
I think that to remove all sys.path.append() calls, we would have to change the directory structure of the project, to be more like pyDARN or pyDARNio. Ultimately, I think this would be a good idea, but something for a future release.